src/common/Navbar.js

```diff
@@ -264,11 +265,13 @@ class Navbar extends Component {
                   </div>
                 </div>
                 <div className="input-group">
+                 {/* <SearchBar
+                />  */}
```

Comment:
Why are we commenting this out here and continuing to use the `AutoComplete` component below?

Reply:
I changed it back.